user.php

```diff
@@ -0,0 +1,108 @@
+<?
+
+
+class User{
+
+	public $db;
+
+	private $states = [
+		'NSW',
+		'ACT',
+		'VIC',
+		'QLD',
+		'TAS',
+		'WA'
+	]
+
+	function __construct()
+	{
+		$this->db = new mysqli('127.0.0.1', 'root', 'password', 'users');
+
+	}
+
+	function check_Postcode($postcode) {
+		//.... call some external api
+		$res = external_api_call();
```

Comment:
Another good example where dependency injection could be used. Instead of calling this function here, there should be an adapter class for this API being injected as a dependency in this class.